packages/block-editor/src/hooks/layout.js

```diff
@@ -254,7 +255,7 @@ function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
 					) }
 				</PanelBody>
 			</InspectorControls>
-			{ ! inherit && layoutType && (
+			{ allowEditing && ! inherit && layoutType && (
```

Comment:
Is this line still needed? It looks like this function returns early if `allowEditing` is falsy.

Reply:
Thanks for spotting that! Leftover from my previous changes here, should be fine to remove.